pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "fool-ai-detector"
+version = "0.0.1"
+authors = [
+    { name="Raphael Weber"},
+    { name="Max Buchholz"},
+    { name="Lukas Müller"},
+    { name="Levi Otterbach"},
+]
+description = "Fooling ai detectors"
+readme = "README.md"
+requires-python = ">=3.8"
```

Comment:
I wouldn't do that. If you set it to 3.11 only, this package would fail to be installed on every other environment. Imagine we would need to execute this package on Google Colab, then we need to use the preinstalled python environment there.

Reply:
yeah same thought and for the point being @EhrenbruderRoschtbraten and I are still using 3.9